src/Components/SignUp/Verification/index.jsx

```diff
@@ -72,6 +74,19 @@ const Verification = () => {
         nonce: details.verification,
       })
       .then((response) => {
+        dispatch({
+          type: "auth/set_session",
+          payload: {
+            user: response.data['user_info'],
+            jwt: {
+              jwt_access_token: response.data['jwt_access_token'],
+              jwt_id_token: response.data['jwt_id_token'],
+              jwt_refresh_token: response.data['jwt_refresh_token'],
+            }
+          },
```

Comment:
Can we create a mapping method which will map the response data to auth payload and we can reuse that method in other places